tests/unit/test_tenant.py

```diff
@@ -12,30 +11,22 @@ def setup(self):
         self.custom_404_msg = "Tenant not found"
         self.schema = TenantSchema
 
-    def test_only_active_lease_returned(self, faker, create_lease, create_tenant):
-        tenant = create_tenant()
-
-        # Create an expired lease for the tenant
-        lease_expired_end = faker.date_time_this_decade(
-            before_now=True, after_now=False
-        )
-        lease_expired_start = lease_expired_end - datetime.timedelta(days=365)
-        lease_expired = create_lease(
-            tenant=tenant,
-            dateTimeStart=lease_expired_start,
-            dateTimeEnd=lease_expired_end,
-        )
-
-        # Create an active lease for the tenant
-        lease_active = create_lease(
-            tenant=tenant, property=PropertyModel.find(lease_expired.propertyID)
-        )
-
-        # Active lease should be the only one that shows up
-        assert tenant.json()["lease"] == lease_active.json()
-
 
 @pytest.mark.usefixtures("empty_test_db")
-class TestTenantFactory:
-    def test_create_tenant(self, create_tenant):
-        assert create_tenant()
```

Comment:
This is testing the `create_tenant` factory and should not be removed.

Reply:
This test is kind of crappy. It doesn't appear to be doing much. But I kinda like having factory tests. If something changes that causes the factory to fail and stop it from performing its job then we would get a lot of red tests, but seeing this test fail would point the developer to the root cause of the issue. While not entirely needed, it can be useful and is cheap to have. The user factory tests are a bit more involved as they test that the factory creates the right role too.